packages/android_alarm_manager_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.8
+
+- Add integration tests.
+
```

Comment:
No need to do a new release for adding integration tests, you can remove this if you want

Reply:
Yes, let's remove it as we definitely not releasing this change as separate version.

Reply:
Thanks for that! This has been removed.